static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -104,9 +121,11 @@ function WidgetViewerModal(props: Props) {
           <Button type="button" onClick={() => undefined}>
             {t('Edit Widget')}
           </Button>
-          <Button priority="primary" type="button" onClick={() => undefined}>
-            {openLabel}
-          </Button>
+          <Link to={path}>
+            <Button priority="primary" type="button" onClick={() => undefined}>
+              {openLabel}
+            </Button>
+          </Link>
```

Comment:
```suggestion
            <Button to={path} priority="primary" type="button">
              {openLabel}
            </Button>
```

Reply:
👍  I originally thought we had to use `Link` to allow command clicking to open in a new tab, but it looks like this also works with `Button`! Updated, thanks!